new-packages/Core/src/Io/File.candy

```diff
@@ -0,0 +1,66 @@
+use ...Bool
+use ...Collections
+use ...Hash
+use ...Maybe
+use ...Operators
+use ...String
+
+## Note: All definitions inside this module are temporary until we have a proper cross-platform file
+## system abstraction in pure Candy.
+
+public trait FileSystemNode: Equals & Hash {
+  fun path(): Path
+
+  fun doesExist(): Bool
```

Comment:
```suggestion
  fun exists(): Bool
```

Reply:
https://dart.dev/guides/language/effective-dart/design#prefer-a-non-imperative-verb-phrase-for-a-boolean-property-or-variable